src/wp-includes/html-api/class-wp-html-open-elements.php

```diff
@@ -226,7 +266,31 @@ public function has_element_in_specific_scope( string $tag_name, $termination_li
 	/**
 	 * Returns whether a particular element is in scope.
 	 *
+	 * > The stack of open elements is said to have a particular element in
+	 * > scope when it has that element in the specific scope consisting of
+	 * > the following element types:
+	 * >
+	 * >   - applet
+	 * >   - caption
+	 * >   - html
+	 * >   - table
+	 * >   - td
+	 * >   - th
+	 * >   - marquee
+	 * >   - object
+	 * >   - template
+	 * >   - MathML mi
+	 * >   - MathML mo
+	 * >   - MathML mn
+	 * >   - MathML ms
+	 * >   - MathML mtext
+	 * >   - MathML annotation-xml
+	 * >   - SVG foreignObject
+	 * >   - SVG desc
+	 * >   - SVG title
+	 *
 	 * @since 6.4.0
+	 * @since 6.7.0 Full implementation.
```

Comment:
Foreign content is not covered yet.
```suggestion
	 * @since 6.7.0 Implementation covering all HTML elements.
```

Reply:
good thought: clarified in 0c9c40f26c8c8749cadf1563cb51357d12852898